src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Ball {
+    private Colors color;
+    private int number;
+
+    public Ball(Colors color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{color='" + color + "', number=" + number + "}";
+    }
```

Comment:
The 'toString' method is correctly overridden, but consider using 'name()' instead of the implicit 'toString()' when concatenating the 'color' enum, to ensure the intended string representation is used.